editor/src/components/canvas/controls/select-mode/flex-gap-controls.tsx

```diff
@@ -53,6 +54,13 @@ export const FlexGapControls = React.memo(() => {
     [dispatch, canvasScale, roundedCanvasOffset],
   )
 
+  const completeDrag = React.useCallback(
+    (event: React.MouseEvent) => {
+      dispatch([CanvasActions.clearDragState(true)], 'canvas')
```

Comment:
why is this mouseup not caught by the canvas handleCanvasEvent?

Reply:
Felt like the entire lifecycle should live in the strategy.